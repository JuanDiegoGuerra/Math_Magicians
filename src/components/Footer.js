import styles from '../styles/Footer.module.css';

const Footer = () => (
  <footer className={styles.footer}>
    <h3 className={styles.copy}>Â© 2023 JD Guerra. All rights reserved.</h3>
  </footer>
);
export default Footer;
