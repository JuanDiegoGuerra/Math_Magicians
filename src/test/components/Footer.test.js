import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('The Footer', () => {
  it('Matches the snapShot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Renders content correctly', () => {
    render(<Footer />);
    expect(screen.getByRole('heading', {
      name: 'Â© 2023 JD Guerra. All rights reserved.',
    })).toBeInTheDocument();
  });
});
