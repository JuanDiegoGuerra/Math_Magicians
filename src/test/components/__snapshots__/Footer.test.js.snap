// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer Matches the snapShot 1`] = `
<footer
  className="footer"
  data-testid="footer-element"
>
  <h3
    className="copy"
  >
    Â© 2023 JD Guerra. All rights reserved.
  </h3>
</footer>
`;
